package.json

```diff
@@ -131,6 +133,8 @@
   },
   "dependencies": {
     "@apollo/client": "^3.7.2",
+    "@babel/preset-env": "^7.22.7",
```

Comment:
These should all be added to `devDependencies`, not `dependencies`, because they are only needed as part of development.